config/initializers/permissions.rb

```diff
@@ -313,25 +313,29 @@
                      {},
                      permissible_on: %i[work_package project],
                      require: :loggedin,
-                     dependencies: :view_work_packages
+                     dependencies: :view_work_packages,
+                     visible: -> { OpenProject::FeatureDecisions.comments_with_restricted_visibility_active? }
```

Comment:
Nice, I didn't know this was a thing 👍🏾

Reply:
I found out on this file when looking for other feature flags :sweat_smile: